tests/situations/commun/vues/defi.test.js

```diff
@@ -488,7 +488,7 @@ describe("La vue d'un défi", function () {
         ];
         question.score = 1;
         vue = composant(question);
-        const reponse = 'tempsFutur';
+        const reponse = { reponse: 'tempsFutur', succes: true, score: undefined };
```

Comment:
heu… Je suis à peu près sûr que ça ne marche pas. Tu ne peux pas changer si facilement le type de `this.reponse` dans cette classe.

Reply:
Ah… je crois comprendre. L'erreur initiale avait été de présumer que le type de réponse était une simple chaine de caractère.